cli/integration/tests/waiter/test_cli.py

```diff
@@ -1272,3 +1272,30 @@ def test_show_service_current(self):
         finally:
             util.delete_token(self.waiter_url, token_name_1, kill_services=True)
             util.delete_token(self.waiter_url, token_name_2, kill_services=True)
+
+    def __test_create_update_token_admin_mode(self, action):
+        cpus = 0.1
+        run_as_user = 'FAKE_USERNAME'
+        token_name = self.token_name()
+        temp_env = os.environ.copy()
+        temp_env["WAITER_ADMIN"] = 'true'
+        action_flags = f'--cpus {cpus} --admin --run-as-user {run_as_user}'
+        if action == 'create':
+            cp = cli.create(self.waiter_url, token_name, flags="-v", create_flags=action_flags, env=temp_env)
+        elif action == 'update':
+            cp = cli.update(self.waiter_url, token_name, flags="-v", update_flags=action_flags, env=temp_env)
+        self.assertEqual(0, cp.returncode, cp.stderr)
+        try:
+            token_data = util.load_token(self.waiter_url, token_name)
+            self.assertIn('update-mode=admin', cli.stderr(cp))
+            self.assertIn(f'Attempting to {action} token in ADMIN mode', cli.stdout(cp))
+            self.assertEqual(cpus, token_data['cpus'])
+            self.assertEqual(run_as_user, token_data['run-as-user'])
+        finally:
+            util.delete_token(self.waiter_url, token_name)
+
+    def test_create_token_admin_mode(self):
+        self.__test_create_update_token_admin_mode('create')
+
+    def test_update_token_admin_mode(self):
+        self.__test_create_update_token_admin_mode('update')
```

Comment:
The update should have an already created token from some other user and then try to update it.